docs/RESOURCES.md

```diff
@@ -0,0 +1,7 @@
+## Recommended Learning Resources
+
+* The [#haskell](http://webchat.freenode.net/?channels=haskell) IRC channel on irc.freenode.net whose goal is to "encourage learning and discussion of Haskell, functional programming, and programming in general". More information about the IRC channel can be found on [the Haskell Wiki page for the channel](https://wiki.haskell.org/IRC_channel).
+* [Hoogle](https://www.haskell.org/hoogle/) is a Haskell API search engine. This allows you to look up what a function does if all you know is its name. Perhaps more interestingly, it also supports searching by type signature: If you want to know what functions have the type signature [(a -> b) -> [a] -> [b]](https://www.haskell.org/hoogle/?hoogle=%28a+-%3E+b%29+-%3E+[a]+-%3E+[b]), Hoogle helps you find out!
```

Comment:
Ah no it didn't. It was just I have a browser extension that thinks this link is an XSS attempt, so it takes out the parentheses and brackets (making the link not quite as useful as it should be). Probably not a problem for most people.